### ISSUE TITLE
Request to eliminate the pre-push hook

### ISSUE THREAD

**Issue by sushanyadav** [Aztriltus/nextjs-ts-tailwind-template#35]
The current pre-push hook in our repository seems to be slowing down our development workflow hindering developers' productivity. We have GitHub Actions for CI checks, so the presence of the **pre-push** hook appears redundant.

Although I am personally okay with keeping **pre-commit** hooks for linting and formatting, I remain uncertain about waiting for few minutes for those checks to complete. I still believe that IDE extensions/defaults can effectively manage these.

This might be a personal preference, so if you have a different view, feel free to share.

Thank you !

**Comment by jeraldgan** [comment]
I agree with @sushanyadav. If there’s a need to fix lint and formatting issues, I usually fix it after I am done with the PR so I think eliminating the pre push hook makes sense to speed up development workflow

**Comment by Aztriltus** [comment]
Yea I think that sounds good
